lib/expressView.js

```diff
@@ -20,7 +20,30 @@ var format = require('util').format;
 var inherit = require('util').inherits;
 var debug = require('debug')(require('../package').name);
 
-var View = util.safeRequire('express/lib/view');
+var ExpressView = util.safeRequire('express/lib/view');
+
+function View(name, options) {
+  options = options || {};
+  this.name = name;
+  this.root = options.root;
+  var engines = options.engines;
+  this.defaultEngine = options.defaultEngine;
+  console.log(this.defaultEngine);
+
+  //var ext = this.ext = extname(name);
+  var ext = this.ext = '.' + this.defaultEngine;
+
+  console.log(this.ext);
+  if (!ext && !this.defaultEngine) {
+    throw new Error('No default engine was specified and no extension was provided.');
+  }
+
+  //if (!ext) name += (ext = this.ext = ('.' != this.defaultEngine[0] ? '.' : '') + this.defaultEngine);
+  this.engine = engines[ext] || (engines[ext] = require(ext.slice(1)).__express);
+  this.path = this.lookup(name);
+}
+
```

Comment:
can you please explain what lines you changed or what is different when compared to the existing ExpressView? thanks